claude-code.el

```diff
@@ -674,6 +681,8 @@ SWITCHES are optional command-line arguments for PROGRAM."
   (let* ((vterm-shell (if switches
                           (concat program " " (mapconcat #'identity switches " "))
                         program))
+         (vterm-environment (cons (format "CLAUDE_BUFFER_NAME=%s" buffer-name)
```

Comment:
Do we need to set `vterm-environment` and `process-environment` above, or is `process-environment` sufficient?